srcpkgs/gammastep/template

```diff
@@ -1,8 +1,9 @@
 # Template file for 'gammastep'
 pkgname=gammastep
-version=2.0.8
+version=2.0.9
 revision=1
-wrksrc="${pkgname}-v$version"
+_ref="v$version"
```

Comment:
I don't think introduce `_ref` is good idea.

Reply:
I would appreciate an explanation as to why. Underscore variables are simple and improve readability. Furthermore this improves my life as a maintainer, because I can easily test an upstream merge request.

Reply:
>for commit, you can't use it with distfiles anyway

Not sure what you mean, as I am quite sure I have been able to change a single line for any ref (branch, commit, or tag) versus having to edit three different occurrences to `v$version` without the _ref variable (as shown below). Are you saying that the distfiles will be wrong for the version defined? If so, I do acknowledge that, but I have had no trouble running clean, then re-genning the checksum in those cases.

```diff
@@ -2,7 +2,7 @@
 pkgname=gammastep
 version=2.0.9
 revision=1
-wrksrc="${pkgname}-v${version}"
+wrksrc="${pkgname}-eab8be4fc33e4c4b953252e2a4f24e4f5322b0b1"
 build_style=gnu-configure
 configure_args="--enable-gui --enable-drm --enable-vidmode --enable-randr
  --enable-wayland --enable-apparmor --with-systemduserunitdir=no
@@ -16,8 +16,8 @@ maintainer="Cameron Nemo <cam@nohom.org>"
 license="GPL-3.0-or-later"
 homepage="https://gitlab.com/chinstrap/gammastep"
 changelog="https://gitlab.com/chinstrap/gammastep/raw/master/NEWS.md"
-distfiles="https://gitlab.com/chinstrap/gammastep/-/archive/v${version}/${pkgname}-v${version}.tar.gz"
-checksum=bbb9d90e1cf30920c1017db4ce5c4652e4c7843fd4c4e34164d99ecbc3bbb4c0
+distfiles="https://gitlab.com/chinstrap/gammastep/-/archive/eab8be4fc33e4c4b953252e2a4f24e4f5322b0b1/${pkgname}-eab8be4fc33e4c4b953252e2a4f24e4f5322b0b1.tar.gz"
+checksum=b3a2fb29c076d1af9ffbb6ccc2a32667954ea22cdc23a90747b78355225e43c8
 build_options="geoclue2"
 build_options_default="geoclue2"
```

Regardless, I have reverted the change for now so as not to introduce inconsistencies or bad samples into the ports tree.